src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -8,18 +8,18 @@
  */
 public class DriverManager {
 
-	private final MacroProxyDriver currentDriver = new MacroProxyDriver();
+	private final Job2dDriver currentDriver = new ProxyDriver();
 
 	/**
 	 * @param driver Set the driver as current.
 	 */
 	public synchronized void setCurrentDriver(Job2dDriver driver) {
-		currentDriver.setDriver(driver);
+		((ProxyDriver)currentDriver).setDriver(driver);
```

Comment:
Unfortunately, I do not allow casting if not necessary. Please find another way to enable recording a macro.